Content.Server/Power/Pow3r/BatteryRampPegSolver.cs

```diff
@@ -54,7 +56,11 @@ public void Tick(float frameTime, PowerState state, IParallelManager parallel)
                 // suppliers + discharger) Then decide based on total layer size whether its worth parallelizing that
                 // layer?
                 _networkJob.Networks = group;
-                parallel.ProcessNow(_networkJob, group.Count);
+                IConfigurationManager _cfg = IoCManager.Resolve<IConfigurationManager>();
+                if (_cfg.GetCVar(CCVars.DebugPow3rDisableParallel))
+                    parallel.ProcessSerialNow(_networkJob, group.Count);
```

Comment:
If you actually want to permanently keep the cvar, make it a bool on the field and have the power system update it when the cvar changes instead of using ioc resolve